templates/problem/problem.html

```diff
@@ -126,6 +167,109 @@ <h2 style="color:#393630; display: inline-block">{{ title }}</h2>
     {% endif %}
     <div><a href="{{ url('chronological_submissions', problem.code) }}">{{ _('All submissions') }}</a></div>
     <div><a href="{{ url('ranked_submissions', problem.code) }}">{{ _('Best submissions') }}</a></div>
+
+    {% if not in_contest %}
+        <hr style="padding-top: 0.1em">
+        {% include 'lightbox.html' %}
+    {% endif %}
+
+    {% if can_vote %}
+        <div class="vote-form" id="id_vote_form">
+            {% if has_voted %}
+                <span class="vote-form-info">{{ _('Current Vote:') }}</span>
+                <span class="vote-form-value">{{ voted_points|floatformat }}</span>
+                <br/> <br/>
+                <span class="vote-form-text">{{  _('Change your vote below!') }}</span>
+            {% else %}
+                <span class="vote-form-text">{{ _("Cast your vote on this problem's weightage below!") }}</span>
+            {% endif %}
+            <br/> <br/>
+            <form method="post"> {% csrf_token %}
+                <span class="vote-form-info">{{ _('Points') }}</span>
+                <input class="vote-form-value" type="number" step="any" name="points" id="id_points" value={{ points_placeholder }}>
+                {% if has_errors and 'points' in problem_points_vote_form.errors %}
+                    <br/>
+                    <span class="voting-form-error">{{ _(problem_points_vote_form.errors['points'][0]) }}</span>
+                    <br/>
+                {% else %}
+                    <br/> <br/>
+                {% endif %}
+                <span class="vote-form-info">{{ _('Note') }}</span>
+                <br/>
+                <textarea class="vote-form-text" name="note" id="id_note" rows="6" cols="80">{{ note_placeholder }}</textarea>
+                {% if has_errors and 'note' in problem_points_vote_form.errors %}
+                    <br/>
+                    <span class="voting-form-error">{{  _(problem_points_vote_form.errors['note'][0]) }}</span>
+                    <br/>
+                {% else %}
+                    <br/> <br/>
+                {% endif %}
+                <input type="submit" value="{{  _('Vote') }}" class="vote-button"/>
+                <input type="hidden" name="vote_confirmation"/>
+            </form>
+        </div>
+
+        <a class="form-button" id="id_vote_button">{% if has_voted %}Change vote{% else %}Vote on problem points{% endif %}</a>
+        {% if has_voted %}
+            <form method="post" id="id_delete_vote_form"> {% csrf_token %}
+                <a class="form-button" id="id_delete_vote_button">{{   _('Delete vote') }}</a>
+                <input type="hidden" name="delete_confirmation"/>
+            </form>
+        {% endif %}
+
+        <script defer>
+            //clear default upon the first click of the note box
+            let cleared_default = false
+            const note_field = document.getElementById('id_note')
+            note_field.onclick = function(){
```

Comment:
```suggestion
            note_field.onclick = function () {
```
nit: space between `function` and `()` only when there are no parameters